src/components/layout/LayoutMain.tsx

```diff
@@ -15,12 +15,35 @@ import { useStore } from "@/store/useStore";
 
 export const LayoutMain = ({ children }: { children: ReactNode }) => {
   const { floatNotifications, removeFloatNotification } = useStore();
+  const [floatNotificationOffsetTop, setFloatNotificationOffsetTop] =
+    useState(0);
+  const headerRef = useRef<HTMLDivElement>(null);
 
   // Init tracking
   useEffect(() => {
     initTracking();
   }, []);
 
+  // FloatNotification dynamic position based on header height (with or without banner)
```

Comment:
Getting header height dynamically instead of hard coding, because it was overlapping float notifications.